lib/Modules/ComputeHooks/ComputeHooks.psm1

```diff
@@ -756,8 +809,18 @@ function Invoke-InstallHook {
         Write-JujuWarning "Failed to set power scheme."
     }
     Start-TimeResync
+    
+    $cfg = Get-JujuCharmConfig
+    if ($cfg['format-raw-devices']) {
+        Get-Disk | Where partitionstyle -eq 'raw' | `
```

Comment:
This should be in a separate function we call here. I think there may already be a function that does this.